lib/r2d2/util.rb

```diff
@@ -44,8 +45,10 @@ def verify_mac(mac_key, encrypted_message, tag)
       raise TagVerificationError unless secure_compare(mac, Base64.decode64(tag))
     end
 
-    def decrypt_message(encrypted_data, symmetric_key)
-      decipher = OpenSSL::Cipher::AES128.new(:CTR)
+    def decrypt_message(encrypted_data, symmetric_key, cipher_key_length_bits = 128)
```

Comment:
Should `cipher_key_length_bits` have a default value, since we always calculate it on line 30?

Reply:
same deal here re android_pay